### PR TITLE
Converting Python services to use HTTP instead of UDP

### PR DESCRIPTION
**Pull request by CatGarab**

### CODE REVIEW

**Comment by CatGarab** [comment]
[Update Python `kubos-service` library to use HTTP](https://trello.com/c/NJtwrd0G/250-update-python-kubos-service-library-to-use-http)

File: examples/python-service/config.toml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [example-service.addr]
-ip = "0.0.0.0"
+ip = "192.168.33.10"
```
Comment (jacoffey3) [inline examples/python-service/config.toml]: Why this?

File: libs/kubos-service/kubos_service/config.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 import argparse
 import toml
 
-DEFAULT_IP = "127.0.01"
+DEFAULT_IP = "127.0.0.1"
```
Comment (jacoffey3) [inline libs/kubos-service/kubos_service/config.py]: wow...that went undetected....

File: examples/python-service/config.toml (hunk 1 above)
Comment (CatGarab) [reply examples/python-service/config.toml]: Whoops, that shouldn't have gotten merged

File: libs/kubos-service/kubos_service/config.py (hunk 2 above)
Comment (plauche) [reply libs/kubos-service/kubos_service/config.py]: lol wow

**Pull request merged**